en/identity-server/next/docs/apis/organization-apis/restapis/claim-mgt.yaml

```
@@ -0,0 +1,648 @@
+  title: Claim Management API
+  description: |
+    This document specifies a **Organization level Claim Management RESTful API** for **WSO2 Identity Server**.
+    
+    This API allows retrieving local/external claims and claim dialects. Organizations have the default claims/claim dialtect defined in Identity Server and custom claims used by B2B shared apps.
+  version: v1
+servers:
+  - url: https://{serverUrl}/t/{tenantDomain}/o/api/server/v1
+    variables:
+      serverUrl:
+        default: localhost:9443
+      tenantDomain:
+        default: carbon.super
+security:
+  - OAuth2: []
+
+tags:
+  - name: management
+    description: Secured Admin-only calls
+paths:
+  /claim-dialects/local/claims:
+    get:
+      tags:
+        - management
+      summary: Retrieve local claims
+      description: Retrieve local claims. <br>
+        <br> <b>Scope(Permission) required:</b> `internal_org_claim_meta_view`
+      operationId: getLocalClaims
+      parameters:
+        - name: attributes
+          in: query
+          description: Define only the required attributes to be sent in the response
+            object.  <br><b> This option is not yet supported. </b>
+          schema:
+            type: string
+        - name: limit
+          in: query
+          description: Maximum number of records to return. <br><b> This option is not yet supported. </b>
+          schema:
+            type: integer
+            format: int32
+        - name: offset
+          in: query
+          description: Number of records to skip for pagination. <br><b> This option is not yet supported. </b>
+          schema:
+            type: integer
+            format: int32
+        - name: filter
+          in: query
+          description: Condition to filter the retrieval of records. <br><b> This option is not yet supported. </b>
+          schema:
+            type: string
+        - name: sort
+          in: query
+          description: Define the order by which the retrieved records should be sorted. <br><b> This option is not yet supported. </b>
+          schema:
+            type: string
+      responses:
+        200:
+          description: Local claims.
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/LocalClaimRes'
+        401:
+          description: Unauthorized.
+          content: {}
+        500:
+          description: Internal Server Error.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        501:
+          description: Not Implemented.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+      x-codeSamples:
+        - lang: Curl
+          source: |
+            curl -X 'GET' \
+            'https://localhost:9443/o/api/server/v1/claim-dialects/local/claims' \
```

```suggestion
            'https://{serverUrl}/t/{tenantDomain}/o/api/server/v1/claim-dialects/local/claims' \
```
Shouldn't this be the format of this URL?

---

hardcoding curls because the curl generation is not in the used plugging. Hadcoding default host port to make it clear

t/{tenantDomain} is not added because single tenancy is promoted in IS